apps/src/templates/progress/ProgressBubbleSet.jsx

```diff
@@ -0,0 +1,85 @@
+/**
+ * An ordered set of progress bubbles.
+ */
+
+import React from 'react';
+import Radium from 'radium';
+import ProgressBubble from './ProgressBubble';
+import color from "@cdo/apps/util/color";
+
+const styles = {
+  main: {
+    position: 'relative',
+    display: 'inline-block'
+  },
+  withBackground: {
+    display: 'inline-block',
+    position: 'relative'
+  },
+  background: {
+    height: 10,
+    backgroundColor: color.lighter_gray,
+    position: 'absolute',
+    left: 0,
+    right: 0,
+    marginTop: (ProgressBubble.height - 10) / 2,
+    marginBottom: (ProgressBubble.height - 10) / 2,
+  },
+  backgroundFirst: {
+    left: 15
+  },
+  backgroundLast: {
+    right: 15
+  },
+  bubble: {
+    position: 'relative',
+  }
+};
+
+const ProgressBubbleSet = React.createClass({
+  propTypes: {
+    startingNumber: React.PropTypes.number.isRequired,
+    statuses: React.PropTypes.arrayOf(
+      React.PropTypes.string
+    ).isRequired,
+    urls: React.PropTypes.arrayOf(
+      React.PropTypes.string
+    ).isRequired
```

Comment:
:+1: Sounds good for now.  Personally I'd prefer an `ary.map(n => _.pick(n, ['url', 'status']))` operation somewhere to the lockstep arrays.

If you wanted to convert from the arrays just for the sake of this component I think you'd use `_.zipWith(statuses, urls, (status, url) => ({status, url}))`.